<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicWidget</class>
 <widget class="QWidget" name="GraphicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QStackedWidget" name="graphicWidget">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="pagePChSS">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="GraphicValue" name="graphic_pchss" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageShP1">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="GraphicTime" name="graphic_shp1" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageShP2">
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="GraphicTime" name="graphic_shp2" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicTime</class>
   <extends>QWidget</extends>
   <header location="global">GraphicTime.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphicValue</class>
   <extends>QWidget</extends>
   <header location="global">GraphicValue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
